library/init/lean/compiler/ir/compilerm.lean

```diff
@@ -91,7 +91,8 @@ registerSimplePersistentEnvExtension {
 @[init mkDeclMapExtension]
 constant declMapExt : SimplePersistentEnvExtension Decl DeclMap := default _
 
-def findEnvDecl (env : Environment) (n : Name) : Option Decl :=
+@[export lean_ir_find_env_decl]
+def findEnvDecl (env : @& Environment) (n : @& Name) : Option Decl :=
```

Comment:
@leodemoura I suppose `export` currently does not respect borrow annotations; should it?

Reply:
Yes, it doesn't. Borrowed annotations are only considered for `extern`.

Reply:
Please do not use them in `export` declarations for now.

Reply:
done